CHANGELOG.md

```diff
@@ -7,211 +7,49 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- [#11068](https://github.com/MetaMask/metamask-extension/pull/11068): refactor custom rpc history test to use fixture
-- [#11097](https://github.com/MetaMask/metamask-extension/pull/11097): Fixing account search results ordering
-- [#11083](https://github.com/MetaMask/metamask-extension/pull/11083): Show a notification about BSC Swaps if a user is on the BSC Mainnet
-- [#11088](https://github.com/MetaMask/metamask-extension/pull/11088): Swaps: Add specific error content if Contract data are not enabled on Ledger
-- [#11087](https://github.com/MetaMask/metamask-extension/pull/11087): Swaps: Add more anonymized tracking for HW wallet and HW wallet type
-- [#10959](https://github.com/MetaMask/metamask-extension/pull/10959): Bump ssri from 6.0.1 to 6.0.2
-- [#11078](https://github.com/MetaMask/metamask-extension/pull/11078): @metamask/providers@8.1.1
-- [#11077](https://github.com/MetaMask/metamask-extension/pull/11077): Sync `master` with `develop`
-- bumping package.json -> 9.5.2
-- Merge remote-tracking branch 'origin/develop' into master-sync
-- [#11065](https://github.com/MetaMask/metamask-extension/pull/11065): return early from metaRPCHandler if stream has ended
-- [#11063](https://github.com/MetaMask/metamask-extension/pull/11063): Swaps: Add anonymized tracking for HW wallet and HW wallet type
-- [#11056](https://github.com/MetaMask/metamask-extension/pull/11056): Sync `master` with `develop`
-- Fixing changelog, bumping package.json -> 9.5.1
-- Merge remote-tracking branch 'origin/develop' into master-sync
-- [#11051](https://github.com/MetaMask/metamask-extension/pull/11051): Fix Sentry sourcemaps
-- [#11049](https://github.com/MetaMask/metamask-extension/pull/11049): Use isValidAddress helper instead of ethereumJsUtil.isValidAddress, when appropriate
-- [#11034](https://github.com/MetaMask/metamask-extension/pull/11034): Bump hosted-git-info from 2.5.0 to 2.8.9
-- [#11016](https://github.com/MetaMask/metamask-extension/pull/11016): Group signature transactions
-- [#11009](https://github.com/MetaMask/metamask-extension/pull/11009): Replace uses of `hasOwnProperty.call` with simpler alternatives
-- [#11031](https://github.com/MetaMask/metamask-extension/pull/11031): Fix error behavior of addEthereumChain
-- [#11030](https://github.com/MetaMask/metamask-extension/pull/11030): Update switchEthereumChain unrecognized chain error
-- [#11029](https://github.com/MetaMask/metamask-extension/pull/11029): Migrate version from `_base` manifest to `package.json`
-- [#10905](https://github.com/MetaMask/metamask-extension/pull/10905): rough in switchEthereumChain
-- [#11025](https://github.com/MetaMask/metamask-extension/pull/11025): Fix redirection to the build quotes page from the swaps failed page
-- [#11011](https://github.com/MetaMask/metamask-extension/pull/11011): Update jsconfig excludes array
+- [#10963](https://github.com/MetaMask/metamask-extension/pull/10963): Ensure method of adding contact when contacts exist
```

Comment:
10963, 10955, 10774 were all part of v9.5.0

Reply:
removed